lib/api/v3/projects/projects_api.rb

```diff
@@ -67,7 +67,6 @@ class ProjectsAPI < ::API::OpenProjectAPI
             mount ::API::V3::Projects::UpdateFormAPI
 
             mount API::V3::Projects::AvailableAssigneesAPI
-            mount API::V3::Projects::AvailableResponsiblesAPI
```

Comment:
Whooops, sorry for overlooking and thanks for fixing

Reply:
Yeah, sorry for stepping over the reviewer boundaries here but it was so straight forward that it seemed weird to note it down instead.

What was weird is that the CI didn't run on your commit [d15d38c](https://github.com/opf/openproject/pull/15249/commits/d15d38cc261a78bdbad2b6bb2daf3c9470bf1e14) before otherwise it would have informed you already.